css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,22 @@
+button {
+  font-size: 20px;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding-top: 22px;
+  /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+  padding-bottom: 22px;
+}
+
+button:hover {
+  color: white;
+  background-color: black;
+}
```

Comment:
Don't forget to change the cursor on hover:

---

![image](https://user-images.githubusercontent.com/16143771/151229783-9c7d6e52-ff43-4651-876f-5725daad9a38.png)